libs/utils/utils.js

```diff
@@ -346,3 +346,15 @@ export const throttle = (delay = 250, throttled = () => {}, opts = {}, ...args)
     }
   };
 };
+
+export function hasPropertyCI(data, name) {
+  return Object.keys(data).some((column) => column.toLowerCase() === name.toLowerCase());
+}
+
+export function propertyNameCI(data, name) {
+  return Object.keys(data).find((column) => column.toLowerCase() === name.toLowerCase());
+}
+
+export function propertyValueCI(data, name) {
+  return data[propertyNameCI(data, name)];
```

Comment:
My preference is to change keys to lowercase before we use the data, like Chris P suggested, rather than call these multiple times. I need to create a ticket for that

Reply:
I have to do some key comparison, so I could do that lowercase conversion in this PR

Reply:
Yeah, actually changing keys to lowercase is probably not something we want to do. I can move these out of utils, I was just anticipating using them in both chart.js and line.js

Reply:
Why do you think that's a bad idea? Here's Chris' comment: https://github.com/adobecom/milo/pull/94#discussion_r938407451

If we need them in list.js, we can add them to their own chart-utils file at that time

Reply:
Agree that these should not be in libs/utils/utils.js but in a chart-utils or chart.js for now (chart.js can also export them for use in list.js)

We're probably going to rename libs/utils/utils.js in the near future.  We're all used to putting util funcs into a file like utils, but really this is a "core-utils" file that is for block loading etc.  Since helix doesn't bundle/transpile files we want to keep block specific utils out of this file.

Reply:
@meganthecoder If we lowercase the headers then all of the legends will be lowercase as well.